src/Microsoft.ML.Data/Model/ModelOperationsCatalog.cs

```diff
@@ -33,6 +33,17 @@ internal ModelOperationsCatalog(IHostEnvironment env)
         /// <param name="stream">A writeable, seekable stream to save to.</param>
         public void Save(ITransformer model, Stream stream) => model.SaveTo(_env, stream);
 
+        /// <summary>
+        /// Save the model to the file.
+        /// </summary>
+        /// <param name="model">The trained model to be saved.</param>
+        /// <param name="modelPath">Path where model should be saved.</param>
```

Comment:
(nit) - I wonder if `modelPath` should be changed to `filePath` instead. `modelPath` sounds like it is the path where the model is already.